import-export-cli/cmd/deprecated/exportAPI.go

```diff
@@ -61,7 +61,7 @@ var ExportAPICmdDeprecated = &cobra.Command{
 		utils.Logln(utils.LogPrefixInfo + exportAPICmdLiteral + " called")
 		var apisExportDirectory = filepath.Join(utils.ExportDirectory, utils.ExportedApisDirName)
 
-		cred, err := cmd.GetCredentials(cmd.CmdExportEnvironment)
+		cred, err := cmd.GetCredentials(cmd.ExportEnvironment)
```

Comment:
We should not rename this variable. I saw that this variable was removed from the root.go. We should not do that. 

Since your command space is using ExportEnvironment, try using CmdExportEnvironment. Since it is in root, it is accessible to all the child go files.

Reply:
restored it to the previous state